ts/features/itwallet/presentation/details/screens/ItwPresentationCredentialDetailScreen.tsx

```diff
@@ -224,6 +225,9 @@ export const ItwPresentationCredentialDetail = ({
       <ContentWrapper>
         <VStack space={24}>
           <ItwPresentationAdditionalInfoSection credential={credential} />
+          <ItwPresentationWalletUpgradeMDLDetailsBanner
```

Comment:
We have `ItwPresentationAdditionalInfoSection` which was designed for this purpose. We can add credential specific components there.